lib/ipfs.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Download and upload to storage. Currently implemented with ipfs.
+ */
+
+const IPFS = require('ipfs')
+const ipfs = {}
+
+ipfs.connect = function(cb) {
+  const node = new IPFS()
+  //node.log.enabled = true
+
+  node.on('init', () => {})
+  node.on('ready', () => {})
```

Comment:
if you bind to the events please handle them as well. Otherwise the lines are unnecessary